purchase_open_qty/models/purchase_order.py

```diff
@@ -19,12 +19,14 @@ def _compute_qty_to_invoice(self):
         for line in self:
             if line.product_id.purchase_method == "receive":
                 qty = line.qty_received - line.qty_invoiced
-                if float_compare(qty, 0.0, precision_digits=precision):
-                    line.qty_to_invoice = qty
-                else:
-                    line.qty_to_invoice = 0.0
+                if float_is_zero(qty, precision_digits=precision):
```

Comment:
Put a comment indicating that this is done for avoid floats like 0.00001 that don't match the filter `qty_to_invoice != 0`